pkg/otelutil/otelutil.go

```diff
@@ -77,6 +79,22 @@ func Setup() (shutdown func() error, err error) {
 		return nil, err
 	}
 
+	serviceResource, err := resource.New(context.Background(), resource.WithAttributes(
```

Comment:
Should the context come as a parameter to `Setup` function?

Can the cancellation of the context provided to `resource` have particular meaning? Like asynchronous graceful shutdown?

Reply:
No, the context here really does nothing TBH other than get handed off to this internal notion otel has of "Detectors" which are basically just dynamic ways of internally doing a `WithAttributes` call.